packages/docs/src/guide-option/queries.md

```diff
@@ -431,6 +431,45 @@ export const resolvers = {
 }
 ```
 
+## Headers
+You can use the `setContext` method from `@apollo/client/link/context` to submit your own custom headers or authorization headers i.e., Bearer.
+
+
+```js
+// other imports ...
+import {setContext} from '@apollo/client/link/context'
+
+// createUploadLink can be used as well
+// jaydenseric/apollo-upload-client library for uploading files using apollo
```

Comment:
I don't think file upload should be mentioned at all in vue-apollo docs, it's unrelated